pkg/splunk/test/azureblobclient.go

```diff
@@ -59,7 +59,7 @@ func (c *MockAzureBlobHandler) CheckAzureBlobRemoteDataListResponse(t *testing.T
 	}
 }
 
-//ListApps ... TODO : refine this method in next sprint as part of list and download azure rest apis
+// ListApps returns the bytes containing blobs list in the XML form
```

Comment:
comment is it XML form data or Json data ?

Reply:
It is XML data.